xmodule/modulestore/xml_exporter.py

```diff
@@ -355,8 +355,13 @@ def adapt_references(subtree, destination_course_key, export_fs):
     Map every reference in the subtree into destination_course_key and set it back into the xblock fields
     """
     subtree.runtime.export_fs = export_fs  # ensure everything knows where it's going!
+    # TODO: Remove logging statements after export issue is resolved
```

Comment:
Also add the ticket number here. It is good for reference

Reply:
Ticket number addded